jupyterhub/orm.py

```diff
@@ -222,7 +222,7 @@ class Group(Base):
     __tablename__ = 'groups'
     id = Column(Integer, primary_key=True, autoincrement=True)
     name = Column(Unicode(255), unique=True)
-    users = relationship('User', secondary='user_group_map', backref='groups')
+    users = relationship('User', secondary='user_group_map', back_populates='groups')
```

Comment:
Can you explain this change? `backref` is supposed to be equivalent to two instances of `back_populates`, so I'm not sure why this is here.

Reply:
Changed to `backref`